cfy_manager/status_reporter/status_reporter.py

```diff
@@ -17,40 +17,37 @@
 
 import json
 import logging
-from os import environ
-from os.path import join
 
 import argh
 
+from cfy_manager.utils.common import output_table
 from ..utils import db
 from ..config import config
 from ..utils.systemd import systemd
-from ..utils.files import update_yaml_file
 from ..utils.common import allows_json_format
 from ..utils.install import is_premium_installed
+from ..utils.scripts import get_encoded_user_ids
 from ..logger import get_logger, setup_console_logger
-from ..utils.scripts import run_script_on_manager_venv
-from ..components.restservice.restservice import REST_SECURITY_CONFIG_PATH
+from ..utils.files import update_yaml_file, read_yaml_file
 from ..constants import (
     STATUS_REPORTER,
-    STATUS_REPORTER_DIR,
+    VERBOSE_HELP_MSG,
+    STATUS_REPORTER_TOKEN,
     STATUS_REPORTER_OS_USER,
+    SELECT_USER_TOKENS_QUERY,
+    STATUS_REPORTER_MANAGERS_IPS,
     STATUS_REPORTER_CONFIGURATION_PATH,
-    STATUS_REPORTER_TOKEN,
-    STATUS_REPORTER_MANAGERS_IPS
 )
 from ..components.components_constants import (
     DB_STATUS_REPORTER,
     BROKER_STATUS_REPORTER,
     MANAGER_STATUS_REPORTER
 )
-VERBOSE_HELP_MSG = "Used to give more verbose output."
-SCRIPTS_PATH = join(STATUS_REPORTER_DIR, 'scripts')
 
 logger = get_logger(STATUS_REPORTER)
 
 
-@argh.arg('--managers-ip',
+@argh.arg('--managers-ips',
```

Comment:
Yoni and Mohammed need to know about this change

Reply:
I'll let them know personally

Reply:
Cancelled. I'll create a JIRA for it